server/questionValidator.js

```diff
@@ -61,30 +61,34 @@ exports.validateAttributeFields = function(question,type){
 }
 
 var validateQuestionAttributesByType = function(question, type){
-    var result;
-
-    switch (type) {
-        case common.questionTypes.REGULAR.value:
-            result = regexAttributeValidator(question);
-            break;
-
-        case common.questionTypes.MULTIPLECHOICE.value:
-            result = multipleChoiceAttributeValidator(question);
-            break;
-
-        case common.questionTypes.TRUEFALSE.value:
-            result = trueAndFalseAttributeValidator(question);
-            break;
-
-        case common.questionTypes.MATCHING.value:
-            result = matchingAttributeValidator(question);
-            break;
-
-        default:
-            result = failMsg;
-            break;
-    }
-    return result;
+	var result;
+
+	switch (type) {
+		case common.questionTypes.REGULAR.value:
```

Comment:
convert you tabs to spaces. use 4 spaces in the backend plz

Reply:
my tabs are 4 spaces, not sure why github shows it different

Reply:
You shall convert them all